lib/fishjam_web/controllers/component_controller.ex

```diff
@@ -76,12 +79,18 @@ defmodule FishjamWeb.ComponentController do
       :error ->
         {:error, :bad_request, "Invalid request body structure"}
 
-      {:error, :room_not_found} ->
-        {:error, :not_found, "Room #{room_id} does not exist"}
-
       {:error, :invalid_type} ->
         {:error, :bad_request, "Invalid component type"}
 
+      {:error, :invalid_room_id} ->
```

Comment:
I feel like we could have `fallback_controller` where we would catch all those `generic` errors with invalid_room_id rpc_failed etc.

Reply:
And it could be used in all controllers which use RPC underneath.